### DISCUSSION TITLE
What Does 'node_modules' Refer To In The Licence File?

### DISCUSSION THREAD

**Discussion by WilliamCrabbRenishaw** [lodash/lodash#5917] [General]
The software licensing team at my company has flagged lodash as a caution based on the follow text found at the bottom of the 'LICENSE' file

"Files located in the node_modules and vendor directories are externally
maintained libraries used by this software which have their own
licenses; we recommend you read them, as their terms may differ from the
terms above."

I've gone through each library within the 'vendor' folder and checked its license to ensure we can use lodash within the company but I'm confused as to what is meant by the 'node_modules' directory. I've gone through many version of lodash via Git and I can't find a single directory named 'node_modules'. Is this superfluous text, or am I missing something obvious?

**Comment by maurer2** [comment]
Hello, node_modules folders are usually if ignored and not checked in. You can create the dependencies locally by cloning the repo and running "npm i" or "npm install".

Cheers
